lisa/sut_orchestrator/azure/common.py

```diff
@@ -216,6 +216,7 @@ class AzureNodeSchema:
     vhd_raw: Optional[Union[Dict[Any, Any], str]] = field(
         default=None, metadata=field_metadata(data_key="vhd")
     )
+    original_vhd_path: str = ""
```

Comment:
Don't add it in node schema, it will be exposed as a parameter in runbook, so it's confusing. Move it to AzureNodeContext, so it's only available runtime.

Reply:
Done